package.json

```diff
@@ -44,6 +45,7 @@
     "clipboardy": "3.0.0",
     "compression": "1.7.4",
     "is-port-reachable": "4.0.0",
+    "qrcode-terminal": "0.12.0",
```

Comment:
How much extra size does this add to the package?

Could we add a test please?